### PR TITLE
IVY ML Engineer Intern Contribution

### PR DESCRIPTION
**Pull request by lucrieffel**
# PR Description

## Related Issue

Close #

## Checklist

- [x] Did you add a function? Yes
- [x] Did you add the tests? Yes
- [x] Did you run your tests and are your tests passing? Yes
- [x] Did pre-commit not fail on any check? Yes
- [x] Did you follow the steps we provided?Yes

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

File: ivy/functional/frontends/paddle/stat.py [hunk 1]
```diff
@@ -50,6 +50,27 @@ def numel(x, name=None):
     return ivy.array(prod if prod > 0 else ivy.array(length, dtype=ivy.int64))
```
Comment (AnnaTz) [inline ivy/functional/frontends/paddle/stat.py:52]: Hi @lucrieffel, please follow are PR naming convention described above.

File: ivy/functional/frontends/paddle/stat.py [hunk 2]
```diff
@@ -50,6 +50,27 @@ def numel(x, name=None):
...
 
 
+@with_supported_dtypes({"float32", "float64"})
```
Comment (AnnaTz) [inline ivy/functional/frontends/paddle/stat.py:53]: You will also need to add a test for you function. Please refer to our [frontend testing documentation](https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html) about this.

**Review by AnnaTz — CHANGES_REQUESTED** [review]

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Pull request closed without merging**